docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,180 @@
+version: '3'
+services:
+
+  ##################################################################################
+
+  geoserver:
+    image: geosolutionsit/geoserver:gs-${DOCKER_IMAGE_VERSION}
+    restart: always
+    ports:
+      - '8001:8080'
+    environment:
+      - JAVA_OPTS=-Xms${MEM} -Xmx${MEM} -XX:+UseParallelGC -XX:+UseParallelOldGC -DGEOWEBCACHE_CONFIG_DIR=$${GEOSERVER_DATA_DIR}/gwc -DGEOSERVER_LOG_LOCATION=$${GEOSERVER_HOME}/logs/$${HOSTNAME}-slave.log -DGEOSERVER_NODE_OPTS=id:slave-$${HOSTNAME} -DGEOSERVER_AUDIT_PATH=/var/geoserver/audits/geoserver-${DOCKER_HOSTNAME}-slave-$${HOSTNAME}
+    volumes:
+      - './volumes/geoserver-slaves/data_dir/:/var/geoserver/datadir'
```

Comment:
the volumes definition is not in line with what I wrote in the issue [here](https://github.com/geosolutions-it/support/issues/1030#issue-755490424)